frontend/packages/dev-console/src/components/pipelines/modals/common/types.ts

```diff
@@ -12,7 +12,6 @@ export type PipelineModalFormResource = {
 };
 
 export type CommonPipelineModalFormikValues = FormikValues & {
-  namespace: string;
```

Comment:
@andrewballantyne I did not find a usage or formik field for this field, so I removed it. Missed I something here?!

Reply:
Yup still [being used](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/pipelines/modals/common/PipelineResourceDropdownField.tsx#L18). Please undo.